FrontEnd/scripts/works.js

```diff
@@ -48,16 +48,24 @@ async function displayWork(target, worksCategory) {
     // NOTE: creats the html elements for each project form the backend
     const workFigure = document.createElement("figure");
     const workImage = document.createElement("img");
-    const workFigCaption = document.createElement("figcaption");
+
+    if (!isEditable) {
+      const workFigCaption = document.createElement("figcaption");
+      workFigCaption.textContent = work.title;
+      workFigure.appendChild(workFigCaption);
+    } else {
+      const editionButton = document.createElement("button");
+      editionButton.setAttribute("class", "galleryEditionBtn");
+      editionButton.innerText = "éditer";
```

Comment:
Maybe should we try this out:

```suggestion
      editionButton.value = "éditer";
```